SimG4CMS/Calo/interface/CaloHitID.h

```diff
@@ -37,6 +45,8 @@ class CaloHitID {
   uint16_t theDepth;
   float timeSliceUnit;
   bool ignoreTrackID;
+  bool hasFineTrackID_;
```

Comment:
this boolean member is not needed - it is enough if by default theFineTrackID=-1

Reply:
I've removed the boolean and now just use the method hasFineTrackId() instead (which checks if theFineTrackId is set to -1).

Reply:
unused `hasFineTrackID_` not actually removed